doc/security/specs/identities_and_root_keys/identities_and_root_keys_fig1.svg

```diff
@@ -0,0 +1 @@
0.46875q-0.234375 0.3125 -0.5625 0.578125q-0.328125 0.265625 -0.765625 0.265625q-0.203125 0 -0.546875 -0.015625q-0.34375 -0.015625 -0.5625 -0.015625l-0.453125 2.75q0.46875 0.140625 0.859375 0.203125q0.390625 0.0625 0.90625 0.0625zm23.402588 -13.109375l0 -0.28125q0 -1.78125 -0.296875 -3.375q-0.28125 -1.59375 -0.765625 -2.984375q-0.5 -1.375 -1.15625 -2.515625q-0.65625 -1.140625 -1.40625 -2.03125q-0.734375 -0.890625 -1.515625 -1.515625q-0.78125 -0.625 -1.53125 -0.953125l-0.6875 1.9375q0.390625 0.3125 0.78125 0.734375q0.40625 0.421875 0.78125 0.953125q0.5625 0.8125 1.0625 1.921875q0.5 1.09375 0.859375 2.484375q0.265625 1.125 0.421875 2.453125q0.171875 1.328125 0.171875 2.859375l0 0.34375q0 1.578125 -0.171875 2.9375q-0.15625 1.34375 -0.421875 2.46875q-0.328125 1.21875 -0.75 2.203125q-0.421875 0.984375 -0.9375 1.765625q-0.421875 0.671875 -0.890625 1.171875q-0.453125 0.5 -0.90625 0.84375l0.6875 1.953125q0.75 -0.328125 1.53125 -0.953125q0.78125 -0.625 1.515625 -1.53125q0.75 -0.875 1.40625 -2.015625q0.65625 -1.140625 1.15625 -2.515625q0.484375 -1.390625 0.765625 -2.984375q0.296875 -1.59375 0.296875 -3.375z" fill-rule="nonzero"/><path fill="#000000" fill-opacity="0.0" d="m1361.8273 1091.0786c0 13.136597 -16.141724 18.69458 -32.283447 26.273315c-16.141724 7.5787354 -32.283447 17.178345 -32.283447 34.356567" fill-rule="evenodd"/><path stroke="#000000" stroke-width="1.0" stroke-linejoin="round" stroke-linecap="butt" d="m1361.8274 1091.0786c0 13.136719 -16.141724 18.69458 -32.28357 26.273315c-8.070801 3.7894287 -16.141724 8.083984 -22.194824 13.578491c-3.0266113 2.7471924 -5.548706 5.7944336 -7.314209 9.228516c-0.8826904 1.717041 -1.576416 3.5307617 -2.0491943 5.4521484l-0.032836914 0.13757324" fill-rule="evenodd"/><path fill="#000000" stroke="#000000" stroke-width="1.0" stroke-linecap="butt" d="m1296.3121 1145.558l1.1169434 4.6983643l2.1644287 -4.3171387z" fill-rule="evenodd"/><path fill="#000000" fill-opacity="0.0" d="m1278.1825 1198.7958c0 18.542847 0.01574707 27.817993 0.03149414 37.085693c0.01574707 9.2677 0.03149414 18.528076 0.03149414 37.05603" fill-rule="evenodd"/><path stroke="#000000" stroke-width="1.0" stroke-linejoin="round" stroke-linecap="butt" d="m1278.1825 1198.7958c0 18.542847 0.01574707 27.817993 0.03149414 37.085693c0.00793457 4.633911 0.01574707 9.265869 0.021606445 15.05542c0.0030517578 2.8947754 0.005493164 6.0789795 0.0072021484 9.697388l0.0013427734 6.3032227" fill-rule="evenodd"/><path fill="#000000" stroke="#000000" stroke-width="1.0" stroke-linecap="butt" d="m1276.5924 1266.9379l1.65271 4.5377197l1.6507568 -4.538452z" fill-rule="evenodd"/><path fill="#000000" fill-opacity="0.0" d="m1257.3649 954.3154c0 47.334717 0.44091797 69.99231 0.88183594 94.66943c0.44104004 24.677246 0.881958 51.3739 0.881958 102.7478" fill-rule="evenodd"/><path stroke="#000000" stroke-width="1.0" stroke-linejoin="round" stroke-linecap="butt" d="m1257.3649 954.31537c0 47.33484 0.44091797 69.99237 0.88183594 94.669495c0.22045898 12.338623 0.44091797 25.182129 0.
```

Comment:
This file contains hyperlinks to Google Docs. Is this intended? Or automatically converted by the tools at a later stage?

Reply:
Removed links. Good catch. Done.